public/docs/_examples/architecture/dart/lib/sales_tax_component.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:angular2/core.dart';
+
+import 'sales_tax_service.dart';
+import 'tax_rate_service.dart';
+
+@Component(
+    selector: 'sales-tax',
+    template: '''
+      <h2>Sales Tax Calculator</h2>
+      Amount: <input #amountBox (input)="0" type="number">
```

Comment:
Please revert to `(change)`.  We're not trying to be responsive. We're trying to make a point about how `(someEvent)="0"` is a trick to cause dirty checking when the user enters something.  `(input)` invites the kind of distracting question that Jesus asked.

Please revert `type="number"`. This is a distraction from the point. Again, we're not looking to produce the best app solution here. We're trying to make a narrow point. Let's not add distractions from that point.